data_diff/dbt.py

```diff
@@ -389,3 +389,23 @@ def _cloud_diff(diff_vars: TDiffVars, datasource_id: int, api: DatafoldAPI, org_
 
 def _diff_output_base(dev_path: str, prod_path: str) -> str:
     return f"\n[green]{prod_path} <> {dev_path}[/] \n"
+
+
+def _initialize_events(dbt_user_id: Optional[str], dbt_version: Optional[str], dbt_project_id: Optional[str]) -> None:
+    set_dbt_user_id(dbt_user_id)
+    set_dbt_version(dbt_version)
+    set_dbt_project_id(dbt_project_id)
+    _email_signup()
+
+
+def _email_signup() -> None:
+    if bool_ask_for_email():
+        email_input = Prompt.ask(
+            "\nWould you like to be notified when a new data-diff version is available?\n\nEnter email or leave blank to opt out (we'll only ask once).\n",
+            default="",
+            show_default=False,
+        )
+        email = email_input.strip()
```

Comment:
Hey, @dlawin! Is it possible to add some validation here?
We have some handy e-mail RegExp at our disposal

Reply:
I was using this at one point https://pypi.org/project/email-validator/

Do you think it's worth it if we're going to need to validate on the backend anyways?

Reply:
Thinking about it a bit I don't see a downside if we validate twice. Let me know how this looks now

Reply:
@dlawin As long as we're using the same regex for both validations - we're good